presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestBingTileFunctions.java

```diff
@@ -319,6 +320,11 @@ public void testGeometryToBingTiles()
 
         assertFunction("transform(geometry_to_bing_tiles(ST_Envelope(ST_GeometryFromText('LINESTRING (59.765625 29.84064389983442, 60.2 30.14512718337612)')), 10), x -> bing_tile_quadkey(x))", new ArrayType(VARCHAR), ImmutableList.of("1230301230", "1230301231"));
 
+        // Empty geometries
+        assertGeometryToBingTiles("POINT EMPTY", 10, emptyList());
+        assertGeometryToBingTiles("POLYGON EMPTY", 10, emptyList());
+        assertGeometryToBingTiles("GEOMETRYCOLLECTION EMPTY", 10, emptyList());
```

Comment:
Add a tests for geometry collection that contains empty geometry. Sorry, totally missed that during the first pass.